index.htm

```diff
@@ -122,6 +126,39 @@ <h2>We Make Your Home Alive</h2>
 Begonia
 </div>
 
+<h1>About WaterMi</h1>
```

Comment:
maybe use h2 here instead - because the about is a bit to big. And increase the subsequent headers (below) accordingly (so they appear smaller)